PostProcessing/Runtime/Effects/MultiScaleVO.cs

```diff
@@ -42,6 +42,9 @@ enum Pass
 
         readonly int[] m_Widths = new int[7];
```

Comment:
Looks like m_Widths and m_Heights are no longer used, at least in this file. Are they referenced elsewhere or can they be removed?

Reply:
Yes, you're right. Those can be removed now